Jenkinsfile

```diff
@@ -673,6 +673,7 @@ def withBeatsEnv(boolean archive, Closure body) {
       sh(label: "Install Go ${GO_VERSION}", script: ".ci/scripts/install-go.sh")
       sh(label: "Install docker-compose ${DOCKER_COMPOSE_VERSION}", script: ".ci/scripts/install-docker-compose.sh")
       sh(label: "Install Mage", script: "make mage")
+      setGitConfig()
```

Comment:
NIT: do we want to add those details to the shared library? We did add a similar one for the apm, see the step `setupAPMGitEmail`

Reply:
yep, I saw it, the intention is to make the workaround and extend setupAPGitEmail to make a new one more for Beats